src/ui/public/kuery/suggestions/conjunction.js

```diff
@@ -0,0 +1,22 @@
+const type = 'conjunction';
+
+const conjunctions = {
+  and: `<p>Requires <span class="suggestionItem__callout">both arguments</span> to be true</p>`,
+  or: `<p>Requires <span class="suggestionItem__callout">one or more arguments</span> to be true</p>`
+};
+
+function getDescription(conjunction) {
+  return conjunctions[conjunction];
+}
+
+export function getSuggestionsProvider() {
+  return function getConjunctionSuggestions({ prefix, end }) {
+    if (!prefix.endsWith(' ')) return [];
+    const suggestions = Object.keys(conjunctions).map(conjunction => {
+      const text = `${conjunction} `;
+      const description = getDescription(conjunction);
+      return { type, text, description, start: end, end };
+    });
+    return Promise.resolve(suggestions);
```

Comment:
Wrapping all the suggestion results in a promise isn't necessary. They all get consumed through `getSuggestionsProvider` in `src/ui/public/kuery/suggestions/index.js` anyway, which wraps the results in a promise anyway, so returning a promise of a value doesn't change how it works when it's actually being consumed. These are all "private" by convention, so they aren't used directly.